fastify.js

```diff
@@ -255,13 +261,16 @@ function fastify (options) {
       }
     },
     prefix: {
-      get: function () { return this[kRoutePrefix] }
+      get () { return this[kRoutePrefix] }
     },
     validatorCompiler: {
-      get: function () { return this[kValidatorCompiler] }
+      get () { return this[kValidatorCompiler] }
     },
     serializerCompiler: {
-      get: function () { return this[kSerializerCompiler] }
+      get () { return this[kSerializerCompiler] }
+    },
+    version: {
+      get () { return this[kVersion] }
```

Comment:
I would load the version number dynamically on-demand and then cache it at the top of the file, similar to `lightMyRequest`.

Reply:
Indeed good point

Reply:
As we use `fs` and `path` only for this, it might be interesting to only require them in the `loadVersion` method no?

Reply:
> As we use `fs` and `path` only for this, it might be interesting to only require them in the `loadVersion` method no?

I think so, we did it per `light-my-request`